packages/automation/src/mapping.ts

```diff
@@ -513,12 +539,38 @@ export const defaultCommandTypeMapping: Record<CommandContractType, ParamDefinit
     'address', // debtToken
     'address', // collateralToken
     'bytes32', // operationName
-    'uint256', // execCollRatio
-    'uint256', // targetCollRatio
+    'uint256', // execLtv
+    'uint256', // targetLtv
     'uint256', // minSellPrice
     'uint64', // deviation
     'uint32', // maxBaseFeeInGwei
   ],
+  [CommandContractType.DmaAavePartialTakeProfitCommandV2]: [
+    'address', //positionAddress
+    'uint16', // triggerType
+    'uint256', // maxCoverage
+    'address', // debtToken
+    'address', // collateralToken
+    'bytes32', // operationName
+    'uint256', // execLtv
+    'uint256', // targetLtv
+    'uint256', // excutionPrice
+    'uint64', // deviation
+    'bool', // closeToCollateral
+  ],
+  [CommandContractType.DmaSparkPartialTakeProfitCommandV2]: [
+    'address', //positionAddress
+    'uint16', // triggerType
+    'uint256', // maxCoverage
+    'address', // debtToken
+    'address', // collateralToken
+    'bytes32', // operationName
+    'uint256', // execLtv
+    'uint256', // targetLtv
+    'uint256', // excutionPrice
+    'uint64', // deviation
+    'bool', // closeToCollateral
```

Comment:
So is it `closeToColalteral` or `withdrawToDebt` @halaprix

Reply:
Probably withdrawToDebt